tests/transport/test_requests.py

```diff
@@ -84,44 +86,44 @@ def test_constructor(self):
         assert authed_session.credentials == mock.sentinel.credentials
 
     def test_request_no_refresh(self):
-        mock_credentials = mock.Mock(wraps=MockCredentials())
-        mock_response = make_response()
-        mock_adapter = MockAdapter([mock_response])
+        credentials = mock.Mock(wraps=CredentialsStub())
```

Comment:
That is allowed when you use autospec, but I also want the stub's behavior which is harder to achieve with a plain mock.